test/parallel/test-util-inspect.js

```diff
@@ -14,6 +14,7 @@ assert.equal(util.inspect(null), 'null');
 assert.equal(util.inspect(/foo(bar\n)?/gi), '/foo(bar\\n)?/gi');
 assert.equal(util.inspect(new Date('Sun, 14 Feb 2010 11:48:40 GMT')),
   new Date('2010-02-14T12:48:40+01:00').toISOString());
+assert.equal(util.inspect(new Date('')), (new Date('')).toString());
```

Comment:
I'd make this simply:

```js
assert.equal(util.inspect(new Date('')), 'Invalid Date');
```

there's no reason to create the second date object.